Transporter.IntegrationTests/Tests/MSSQL-MSSQL/MssqlToMssqlIntegrationTests.cs

```diff
@@ -0,0 +1,126 @@
+using System;
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Threading;
+using System.Threading.Tasks;
+using AutoFixture;
+using Dapper;
+using FluentAssertions;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Moq;
+using NUnit.Framework;
+using Transporter.Core.Utils;
+using Transporter.IntegrationTests.Base;
+using Transporter.IntegrationTests.Constants;
+using Transporter.IntegrationTests.Objects;
+using TransporterService;
+
+namespace Transporter.IntegrationTests.Tests.Couchbase_MSSQL
+{
+    public class MssqlToMssqlIntegrationTests : IntegrationTestBase
+    {
+        private Fixture _fixture;
+        private Mock<IDateTimeProvider> _mockDateTimeProvider;
+        
+        [SetUp]
+        public void SetUp()
+        {
+            _fixture = new Fixture();
+            _mockDateTimeProvider = new Mock<IDateTimeProvider>();
+            _mockDateTimeProvider.Setup(m => m.Now).Returns(DateTime.UtcNow.AddMinutes(-5));
+        }
+
+        [Test]
+        public async Task Transporter_ShouldTransportData()
+        {
+            //Arrange
```

Comment:
Can we rename "Arrange" with "Arrange & Act" for readability? What do you think?

Reply:
Changed as "Arrange & Act"